components/molecules/EntryBanner.js

```diff
@@ -0,0 +1,27 @@
+import Banner from "../atoms/Banner";
+import { ActionButton } from "../atoms/ActionButton";
+
+export const EntryBanner = () => {
```

Comment:
A component where text is hard coded is not a component at all. Hard coded text in a component makes i18n very difficult, all text should come from props or there should not be a component at all. I am of the opinion that this component should not exist and instead this should be done in the homepage